Library/Homebrew/downloadable.rb

```diff
@@ -0,0 +1,132 @@
+# typed: true
+# frozen_string_literal: true
+
+require "url"
+require "checksum"
+
+# @api private
+class Downloadable
+  extend T::Sig
+
+  include Context
+
+  sig { returns(T.nilable(URL)) }
+  attr_reader :url
+
+  sig { returns(T.nilable(Checksum)) }
+  attr_reader :checksum
+
+  sig { returns(T::Array[String]) }
+  attr_reader :mirrors
+
+  sig { void }
+  def initialize
+    @mirrors = T.let([], T::Array[String])
```

Comment:
Is the `T.let` needed here?

Reply:
Cool. Like the `::`: if it's not needed, remove, if it is: resolve this and ignore me 😁